### PR TITLE
add toStream()

### PR DESCRIPTION
**Pull request by orangemi**
add function toStream() and make it more like to `stdout` and `stderr`
example:
```js
var sender = new FluentSender(...);
var logger = new Console(sender.toStream('stdout'), sender.toStream('stderr'));
logger.log('...')
```

### CODE REVIEW

**Comment by zensh** [comment]
Good job, it is every useful.

@okkez I expect to use this stream.

**Comment by orangemi** [comment]
Express/koa morgan also use `stream` as `output`
https://github.com/expressjs/morgan#stream
https://github.com/koa-modules/morgan#1x-100-working-with-morgan-and-koa-v2

**Comment by zensh** [comment]
@orangemi @okkez 

ilog, toa-morgan use `stream` too~

https://github.com/teambition/ilog#ilog_stdout
https://github.com/toajs/toa-morgan#stream

File: lib/sender.js [hunk 1]
```diff
@@ -226,6 +227,24 @@ FluentSender.prototype._setupErrorHandler = function() {
...
+  var self = this;
+  var writable = new stream.Writable();
+  var dataString = '';
+  writable._write = function(chunk, encoding, callback){
+    var dataArray = chunk.toString('UTF-8').split(/\n/);
+    function next(cb){
+      if (dataArray.length) dataString += dataArray.shift();
+      if (!dataArray.length) return process.nextTick(cb);
+      self.emit(label, dataString, function(){
+        dataString = '';
```
Comment (zensh) [inline lib/sender.js]: should handle error

**Comment by zensh** [comment]
Hi @okkez Would you accept this PR?

**Comment by okkez** [comment]
Sounds good, but I don't have enough time to review this PR.
I will merge this PR in this week and release new version.

**Comment by okkez** [comment]
Released 2.1.0.

**Pull request merged**